src/blocks/remote-data-empty-result/block.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://schemas.wp.org/trunk/block.json",
+  "apiVersion": 3,
+  "name": "remote-data-blocks/empty-result",
+  "version": "0.1.0",
+  "usesContext": [ "remote-data-blocks/remoteData" ],
+  "title": "Remote Data Empty Result",
```

Comment:
"Empty result" doesn't quite parse to me. We want to display content when there are no results. Can we follow the pattern of the core block and call it "No Remote Data Results"?

Reply:
Agreed, I'll make that change.